### DISCUSSION TITLE
AOE 2 DE - WinVerifyTrust failing (Issue After update 29~ June 23)

### DISCUSSION THREAD

**Discussion by Dnao0** [The-Wineskin-Project/WineskinServer#425] [Ideas]
Hi thought this could be a good thing to add.

Latest AOE2 DE update **broke** the compatibility with Wine (proton, crossOver unofficial, etc). Proton users seems to have a workaround and i found this that could work for this version:

The steps to follow will be:

1. Get the Patch from: [PATCH](https://gitlab.winehq.org/wine/wine/-/merge_requests/3201)
2. Install Patch.
3. Recompile.

Related forums:

- [WINE FORUM](https://bugs.winehq.org/show_bug.cgi?id=55165)
- [STEAM FORUM](https://steamcommunity.com/app/813780/discussions/3/3816284554974767512/?ctp=3#c3806153359221553928)

I may try to do this myself but thought it may benefit all of us Playing AOE2 DE on MacOs using this version of wine.

 (I don't have the full knowledge yet any guide is welcome)

**Comment by Gcenx** [comment]
This one’s on my todo list.

Thouht it’s a first for someone to directly link to the Bugzilla report and MR with a fix that’s always appreciated.

**Reply by Dnao0** [reply]
Nice,
I really wanted to play :D haha

**Comment by Dnao0** [comment]
Hi Gcenx,

is the new 'WS11WineCX64Bit22.1.1-6' with this patch?

**Reply by Gcenx** [reply]
@Dnao0 the patch is part of `WS11WineCX64Bit22.1.1-8` that you should see shortly

**Reply by Dnao0** [reply]
Amazing! Greatly Appreciated
